### PR TITLE
V0.17.0

### PR DESCRIPTION
**Pull request by mikaelene**

### CODE REVIEW

**Comment by swanderz** [comment]
any outstanding work for this?

**Comment by mikaelene** [comment]
This is blocked by https://github.com/fishtown-analytics/dbt/issues/2548

**Comment by alittlesliceoftom** [comment]
It looks like https://github.com/fishtown-analytics/dbt/issues/2548 will be fixed in 0.18.0 . Does that mean no release for 0.17 for dbt-sqlserver?

**Comment by mikaelene** [comment]
Yes. We have to skip 0.17. I am having vacation for a couple of weeks but will look at an 0.18 adapter after that.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by mikaelene** [comment]
@camfrout , this is great news! I'll try to have a test run soon so we can get this going again.

File: dbt/include/sqlserver/macros/materializations/snapshot/snapshot.sql [hunk 1]
```diff
@@ -1,206 +1,4 @@
...
-    {% call statement() %}
-      alter table {{ relation }} add "{{ column.name }}" {{ column.data_type }};
-    {% endcall %}
-  {% endfor %}
-{% endmacro %}
\ No newline at end of file
+{% macro sqlserver__post_snapshot(staging_relation) %}
+  -- Clean up the snapshot temp table
+  {% do drop_relation(staging_relation) %}
+{% endmacro %}
```
Comment (swanderz) [inline dbt/include/sqlserver/macros/materializations/snapshot/snapshot.sql]: why did all of  these macros go away? are they not needed anymore?

**Comment by mikaelene** [comment]
Since v.018.0 is now released I'm closing this

**Pull request closed without merging**